packages/getters/src/value-view.ts

```diff
@@ -40,7 +40,11 @@ export const getValidatorIdentityKeyFromValueView = getValidatorInfoFromValueVie
   getIdentityKeyFromValidatorInfo,
 );
 
-export const getDisplayDenomExponentFromValueView = getMetadata.pipe(getDisplayDenomExponent);
+export const getDisplayDenomExponentFromValueView = createGetter((valueView?: ValueView) =>
```

Comment:
Changes to this file make it possible for these getters to actually be optional. Hopefully this workaround won't be necessary after https://github.com/penumbra-zone/web/issues/1153